pre_install_report/library/pre_install_check.py

```diff
@@ -197,30 +195,30 @@ def check_details(self, kubectl, ingress_port, ingress_host, ingress_controller,
         global_data = self.evaluate_nodes(nodes_data, global_data, cluster_info, quantity_)
 
         params = {}
-        params[viya_constants.INGRESS_CONTROLLER] = ingress_controller
-        params[viya_constants.INGRESS_HOST] = str(ingress_host)
-        params[viya_constants.INGRESS_PORT] = str(ingress_port)
         params[viya_constants.PERM_CLASS] = utils
         params[viya_constants.SERVER_K8S_VERSION] = self._k8s_server_version
         params['logger'] = self.sas_logger
 
         permissions_check = PreCheckPermissions(params)
         self._check_permissions(permissions_check)
 
-        test_list = [viya_constants.INSUFFICIENT_PERMS, viya_constants.PERM_SKIPPING]
```

Comment:
Can you elaborate on why this split of test_list to two different results processed separately going forward.  Comments would be helpful here.

Reply:
Discussed this with @lasiva.   The approach initially taken to split test_list into two different results was re-considered and initial changes reverted.    The minor issue warning message will be addressed separately now.